content/post/2021-12-01-mlr3spatial/index.Rmd

```diff
@@ -0,0 +1,30 @@
+---
+title: mlr3spatial
+author: Marc Becker
+date: '2021-12-01'
+slug: mlr3spatial
+categories:
+  - R
+tags:
+  - mlr3spatial
+  - CRAN
+---
+
+We are happy to announce that [mlr3spatial](https://mlr3spatial.mlr-org.com/) has been released on [CRAN](https://cran.r-project.org/web/packages/mlr3spatial/index.html). 
+[mlr3spatial](https://mlr3spatial.mlr-org.com/) simplifies the handling of spatial objects in the [mlr3](https://github.com/mlr-org) ecosystem.
+Before [mlr3spatial](https://mlr3spatial.mlr-org.com/), the user had to extract raw values from spatial objects to train a model or predict a map.
+
+Now, mlr3 [tasks](https://mlr3.mlr-org.com/reference/Task.html) can directly read from spatial objects via specialized [data backends](https://mlr3.mlr-org.com/reference/DataBackend.html).
+Such tasks can be used to train a model or to perform resampling just like any other mlr3 task.
+We support spatial raster objects created by the packages [terra](https://CRAN.R-project.org/package=terra), [raster](https://CRAN.R-project.org/package=raster) and [stars](https://CRAN.R-project.org/package=stars).
+Additionally, vector data is handled with the package [sf](https://CRAN.R-project.org/package=sf).
+
+The `predict_raster()` function makes a map with predictions from an mlr3 learner.
```

Comment:
We should mention that we use `geom_sf()` under the hood and briefly mention what is happening behind the scenes (i.e. what steps we take away from the user).

PS: "making a map" should be rephrased :)

Reply:
Converting an sf object to data.table?

Reply:
`predict_raster()` does way more, doesn't it?